google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/models/MutationTest.java

```diff
@@ -281,7 +302,12 @@ public void fromProtoTest() {
             ByteString.copyFromUtf8("fake-value"))
         .deleteCells("fake-family", ByteString.copyFromUtf8("fake-qualifier"))
         .deleteFamily("fake-family2")
-        .addToCell("agg-family", "qual1", 1000, 1234);
+        .addToCell("agg-family", "qual1", 1000, 1234)
+        .mergeToCell(
+            "agg-family",
+            "qual2",
+            1000,
+            Base64.getEncoder().encode(BigInteger.valueOf(1234).toByteArray()));
```

Comment:
can we make this the "real" way instead? I worry about people looking at these tests for examples and doing it wrong

Reply:
Done